### PR TITLE
107 add prototype

### LINKED ISSUE #107: Add the reference prototype raw html + css into the repo
Acceptance Criteria:
- [ ] The prototype is available on staging at /prototype
- [ ] The prototype exists in the repo

### PR DESCRIPTION
**Pull request by RpprRoger**
Fixes #107 

## Description

- Add Holly and Matt's efforts in the prototype
- This code will not directly go into production

### CODE REVIEW

File: prototype/howwework/project-entries.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  image: 'howwework/ourwork-momo.jpg',
+  description: `We work with Mind of My Own to help young people participate fully in their care and make it easier to speak up anytime they want. We’ve built apps that make this happen,
+    leading on many aspects of the process.`,
+  problem: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.`,
+  solution: `Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat`,
+  destination: 'https://www.mindofmyown.org.uk',
+})
+
+window.entries.push({
+  title: 'Alexander Rose',
```
Comment (alexalala) [inline prototype/howwework/project-entries.js]: noooooo 😝

Comment (HedgehogHolly) [reply prototype/howwework/project-entries.js]: needs to be changed to Alexandra Rose 😆 👍

Comment (RpprRoger) [reply prototype/howwework/project-entries.js]: This content is changing anyway, https://github.com/neontribe/www/issues/98

**Comment by katjam** [comment]
For what it is worth - my opinion is still that it is not great practice to push 'known defaults' into master.

**Pull request merged**